README.md

```diff
@@ -57,6 +66,9 @@ forced timeouts. FTP Secure (FTPS) is also supported.
   (default: 30). Use 0 to disable the authentication timeout.
 * **passive**: Set to false to prevent a connection in passive mode (default:
   true).
+* **ftps**: Set to true if connecting to a FTP Secure server (default: false).
+* **ftps_mode**: Set to one of the following: DoubleBagFTPS::EXPLICIT or DoubleBagFTPS::IMPLICIT (default: DoubleBagFTPS::IMPLICIT).
```

Comment:
Please wrap all the constants in backticks. I.E. `DoubleBagFTPS::EXPLICIT` or `DoubleBagFTPS::IMPLICIT` (default: `DoubleBagFTPS::IMPLICIT`).